packages/core/src/html/linkProcessor.js

```diff
@@ -22,6 +22,15 @@ const defaultTagLinkMap = {
   script: 'src',
 };
 
+const tagsToValidate = [
```

Comment:
how about a `Set`?

Reply:
Oops, yup we should be using set for this.